apps/browser-extension-wallet/src/views/browser-view/features/multi-wallet/hardware-wallet/steps/Connect.tsx

```diff
@@ -74,7 +75,7 @@ export const Connect: VFC = () => {
   const onRetry = useCallback(() => {
     setDiscoveryState(DiscoveryState.Requested);
     setConnectionError(null);
-    void analytics.sendEventToPostHog(postHogOnboardingActions.hw.CONNECT_HW_TRY_AGAIN_CLICK);
+    void analytics.sendEventToPostHog(postHogActions.hardware.CONNECT_HW_TRY_AGAIN_CLICK);
   }, [analytics]);
```

Comment:
Do we need this in the dependency array?
But why are we wrapping `onRetry` with `useCallback`?

```suggestion
  }, [analytics, postHogActions.hardware.CONNECT_HW_TRY_AGAIN_CLICK]);
```

Reply:
I guess it doesn't have a noticeable impact. If it is already there, let's keep it, WDYT?

Reply:
I don't mind keeping it.